### ISSUE TITLE
hw2 review

### ISSUE THREAD

**Issue by janeyho** [mkcomer237/DAT_SF_16#4]
Hi Matt, 

In item 4 of your python notebook, I'm not sure I understand why you need the cleanline for loop.

for line in rawfile:

```
cleanline = line.strip()
if cleanline == '':
    continue 

tweetfull = json.loads(line.strip())
```

@ghego @Breucopter

**Comment by mkcomer237** [comment]
Yeah it looks like that is redundant - i was originally having trouble because the tweets were spaced out a few lines in the results but I ended up not needing that code.  Also it's Max, not Matt :)

**Comment by janeyho** [comment]
OPPS . sorry, MAX!

**Comment by ghego** [comment]
Congrats on doing HW2!

a couple of comments:
- could you have downloaded the data in the pandas dataframe without the command line pipe? (very nice tool btw)
- do you know the with(... ) python command? If not maybe worth checking out.
- the graph is nice. the Y axis is labeled as "count", are you sure it's a count?
